### PR TITLE
Show how to parse params into stage cmd

### PR DESCRIPTION
**Pull request by RCdeWit**
@alex000kim showed me how to do this for a project in which I'm using Papermill to run an entire notebook from a single stage. As far as I could find, this approach isn't yet covered by the docs. Thought it might be worth highlighting, even though it's not always the most proper approach.

### CODE REVIEW

**Comment by daavoo** [comment]
> As far as I could find, this approach isn't yet covered by the docs. 

It is documented but quite "hidden": https://github.com/iterative/dvc.org/issues/3670

> Thought it might be worth highlighting, even though it's not always the most proper approach.

I think it's a valid and proper approach in many cases, in fact, I consider it a "signature" feature that allows you to track and use parameters in a non-intrusive manner (no need to alter your code to load params as long as you had set up argument parsing)

**Comment by RCdeWit** [comment]
Ah, thanks for pointing that out, @daavoo! In that case, a link to the relevant section would probably suffice here!

File: content/docs/command-reference/stage/add.md [hunk 1]
```diff
@@ -425,6 +425,24 @@ We use [ruamel.yaml](https://pypi.org/project/ruamel.yaml/) which supports YAML
...
+
+```yaml
+stages:
+  train:
+    cmd: python train_model.py ${seed} model.p
+    deps:
+    - train_model.py
+    - matrix-train.p
+    params:
+    - seed
```
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply content/docs/command-reference/stage/add.md:439]: Wait no, I did know this. It's already in that section I mentioned 😬 (as David mentioned).

**Comment by jorgeorpinel** [comment]
> a link to the relevant section would probably suffice here!

➕ @RCdeWit

**Comment by RCdeWit** [comment]
How about an `admon` with a reference like this? 😄

File: content/docs/command-reference/stage/add.md [hunk 2]
```diff
@@ -425,6 +425,14 @@ We use [ruamel.yaml](https://pypi.org/project/ruamel.yaml/) which supports YAML
 
 </admon>
 
+<admon type="tip">
+
+You can also [use templating]
+(https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#templating) to
+parse parameters directly from `params.yaml` into the pipeline stage.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/stage/add.md:432]:
```suggestion
You can also [use templating] to parse parameters directly from `params.yaml`
into the stage.

[use templating]: /doc/user-guide/project-structure/dvcyaml-files#templating
```

**Review by jorgeorpinel — APPROVED** [review]
LGTM 🙂

**Pull request merged**